mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -4557,6 +4557,83 @@ resource "google_container_cluster" "with_private_endpoint_subnetwork" {
 `, containerNetName, s1Name, s1Cidr, s2Name, s2Cidr, clusterName)
 }
 
+func TestAccContainerCluster_withPrivateClusterConfigPrivateEndpointSubnetwork(t *testing.T) {
+	t.Parallel()
+
+	r := acctest.RandString(t, 10)
+
+	subnet1Name := fmt.Sprintf("tf-test-container-subnetwork1-%s", r)
+	subnet1Cidr := "10.0.36.0/24"
```

Comment:
Can we bake this & subnet2Cidr into the template? Values that don't need a random suffix or to be changed in an update test generally should be.

Reply:
I removed the `subnet1Cidr` and `subnet2Cidr` variables and function arguments. Please let me know if this is what you had in mind.

Reply:
Yep!